Apps/phsignalfx/signalfx.json

```diff
@@ -0,0 +1,1214 @@
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.results.*.value",
+                    "data_type": "string",
+                    "example_values": [
+                        "i-0872664e6358a5483_us-west-2_102730102752"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.results.*.created",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1612448904607
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.results.*.creator",
+                    "data_type": "string",
+                    "example_values": [
+                        "EtJfNWZAEAE"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.results.*.description",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.results.*.lastUpdated",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1614953320349
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.results.*.lastUpdatedBy",
+                    "data_type": "string",
+                    "example_values": [
+                        "AAAAAAAAAAA"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.results.*.customProperties.aws_arn",
+                    "data_type": "string",
+                    "example_values": [
+                        "arn:aws:ec2:us-west-2:102730102752:instance/i-0872664e6358a5483"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.results.*.customProperties.aws_state",
+                    "data_type": "string",
+                    "example_values": [
+                        "{Code: 80,Name: stopped}"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.results.*.customProperties.aws_region",
+                    "data_type": "string",
+                    "example_values": [
+                        "us-west-2"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.results.*.customProperties.aws_image_id",
+                    "data_type": "string",
+                    "example_values": [
+                        "ami-0bc06212a56393ee1"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.results.*.customProperties.aws_tag_Name",
+                    "data_type": "string",
+                    "example_values": [
+                        "Phantom"
```

Comment:
Update it with the dummy value like "Tag Name"

Reply:
Fixed